packages/plugin-core/src/commands/pods/BaseExportPodCommand.ts

```diff
@@ -179,21 +180,16 @@ export abstract class BaseExportPodCommand<
             break;
           }
           case PodExportScope.Note: {
-            const promises = [];
```

Comment:
Removing this pattern leads to an ESLint warning about having await inside a for loop.

Reply:
that's why I added a es-lint ignore  at the top of the file
`/* eslint-disable no-await-in-loop */`

Reply:
But why change it if the current pattern doesn't warn?